uk.co.compucorp.civicrm.tasksassignments/js/test/crm-tasks-workflows/controllers/casetype.extended.controller.spec.js

```diff
@@ -23,14 +36,43 @@
     }));
 
     describe('when initialized', function () {
-      var expectedValue;
+      var expectedApiCalls;
 
       beforeEach(function () {
-        expectedValue = { actTypes: { values: getActivityTypes() } };
+        expectedApiCalls = { actTypes: { values: getActivityTypes() } };
       });
 
       it('initialises the parent controller with activity types having component labels', function () {
-        expect($log.debug).toHaveBeenCalledWith('Init CaseTypeCtrl with', jasmine.any(Object), jasmine.any(Object), expectedValue);
+        expect(parentControllerSpy).toHaveBeenCalledWith(expectedApiCalls);
+      });
+    });
+
+    describe('when creating a new case type', function () {
+      beforeEach(function () {
+        $scope = $rootScope.$new();
+
+        initController($scope);
+      });
+
+      it('it clears all activities from the main activity set', function () {
```

Comment:
I don't think we add "it", so `it('clears all activities from the main activity set'` should be enough. Same for other places.

Reply:
you you are right.